### ISSUE TITLE
Ownable is not implemented in Wizard. TL;DR the core features like "transfer" require access control by default.

### ISSUE THREAD

**Issue by tpmccallum** [OpenZeppelin/contracts-wizard#80]
Hi,

Just a quick observation from my experience with the Wizard (which is awesome by the way).

If a user selects the `Ownable` radio button in the `Access Control` section, the smart contract source code does not actually implement any ownable functionality. For example, the following code is generated.

**Example 1**
```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.2;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract MyToken is ERC20 {
    constructor() ERC20("MyToken", "MTK") {}
}
```

Instead, the code should look more like the following.

**Example 2**
```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.2;

import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
import "@openzeppelin/contracts/access/Ownable.sol";

contract MyToken is ERC20, Ownable {
    constructor() ERC20("MyToken", "MTK") {}
}
```

If the code from example 1 is deployed anyone can send tokens to anyone and so forth. There is not concept of ownership implemented.

This can be tested by deploying the code from example 1 and then instantiating an ERC20 contract instance using:
- the contract address from example 1's deployment
- the ABI from example 2's compile

Querying the owner will return a zero address `0x0...00` and any functions where the `onlyOwner` modifier should be implemented i.e. `transfer` will be callable by anyone and still succeed.

Hope this makes sense :)
Thanks

**Comment by frangio** [comment]
Thanks for bringing this up! I've noticed it as well.

The issue is that toggling "Ownable" doesn't really mean "add Ownable" but "add Ownable if any of the above selected features require access control".

The design definitely doesn't convey that.

I'm thinking the access control section should be more like the upgradeability section where there is a toggle at the top and if enabled then there are the two choices.

**Comment by tpmccallum** [comment]
Hi @frangio 
Ah, I see. So for example when I click "mintable" it automatically adds "ownable" because the "ownable" radio button is selected.

Yes I completely agree, there could be a toggle beside the heading "access control".

![Screen Shot 2022-01-18 at 9 02 07 am](https://user-images.githubusercontent.com/9831342/149845954-ad523a68-d9df-4247-ab5a-3a71f5b4f5f6.png)

If a user switches the toggle on then the user has the choice of either "ownable" or "roles" (the default being "ownable").

As the Wizard currently sits, a project and/or user may deploy their ERC20 contract with the following settings (thinking that "ownable" is implemented somewhere in the inheritance).

![Screen Shot 2022-01-18 at 9 09 38 am](https://user-images.githubusercontent.com/9831342/149846271-1515f39a-2520-41f8-86e0-1cec2a763631.png)

**The project will then be surprised to find out that any external account address can instantiate their contract and then easily transfer all of the projects tokens out of the projects ERC20 contract.**

**Comment by frangio** [comment]
But I don't really understand why you mention transfer. Ownable is not meant to affect the transfer function. Transfer is native to ERC20 and anyone with balance can transfer their own tokens.

What is exactly the thing a project would be surprised with?

**Comment by tpmccallum** [comment]
Hi @frangio 
You are 100% correct. A thousand apologies.

I'll tell you exactly what happened. When I refreshed my remix session, in my browser, the environment defaulted to "JavaScript VM (London)", with out me noticing. I intended to have the environment connected to Ropsten via MetaMask at all times.

I seemed to be able to transfer ERC20 tokens on the contract I deployed; even when changing between external accounts in MetaMask. **This seemed very odd to me and I was concerned that there was some underlying issue**. Now I understand that remix was not even acknowledging MetaMask and I was transacting as the same single account in the JavaScript VM (London) environment. Again, a thousand apologies and sorry for taking up your valuable time.

As you say the transfer function is fundamentally only able to transfer tokens if the msg.sender actually has tokens to transfer. 

I have to own, both, my correctness and my blunders, right? :) 
Thanks again for your time.
Kind regards
Tim
